### PR TITLE
Install Pandas and other data science python libs

### PR DESCRIPTION
**Pull request by jashapiro**
### Purpose/implementation Section

Add pandas and other related packages to the docker.

#### What scientific question is your analysis addressing?

None yet, but needed for later.

#### What was your approach?

pip install with explicit versions for all installed and dependency packages.

#### What GitHub issue does your pull request address?

N/A

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.

Does the install method used seem reasonable?

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -157,5 +157,15 @@ RUN apt-get update -qq && apt-get -y --no-install-recommends install \
...
+RUN pip3 install "numpy==1.17.3" && \
+   pip3 install "six==1.13.0" "setuptools==41.6.0" && \
+   pip3 install "cycler==0.10.0" "kiwisolver==1.1.0" "pyparsing==2.4.5" "python-dateutil==2.8.1" "pytz==2019.3" && \
+   pip3 install "matplotlib==3.0.3" && \
+   pip3 install "scipy==1.3.2" && \
+   pip3 install "pandas==0.25.3"
+
 #### Please install your dependencies here
-#### Add a comment to indicate what analysis it is required for
+#### Add a comment to indicate what analysis it is required for
```
Comment (cgreene) [inline Dockerfile]:
```suggestion
#### Add a comment to indicate what analysis it is required for

```

File: Dockerfile [hunk 2]
```diff
@@ -157,5 +157,15 @@ RUN apt-get update -qq && apt-get -y --no-install-recommends install \
...
+# Install python3 data science basics (pandas)
+# using pip to get more current versions
+RUN apt-get update -qq && apt-get -y --no-install-recommends install python3-pip 
+RUN pip3 install "numpy==1.17.3" && \
+   pip3 install "six==1.13.0" "setuptools==41.6.0" && \
+   pip3 install "cycler==0.10.0" "kiwisolver==1.1.0" "pyparsing==2.4.5" "python-dateutil==2.8.1" "pytz==2019.3" && \
+   pip3 install "matplotlib==3.0.3" && \
+   pip3 install "scipy==1.3.2" && \
+   pip3 install "pandas==0.25.3"
+
```
Comment (cgreene) [inline Dockerfile]: Did you want to set the default python to python3 or should we remind folks to use python3 in their shell scripts/CI sections if they want to run with it?

Comment (cgreene) [reply Dockerfile]: Actually - now that I'm typing this out I think just reminding folks to use the `python3` command is the way to go.

Comment (jashapiro) [reply Dockerfile]: I decided not to change it; I was worried about breaking other things in strange ways.

**Pull request merged**